CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Karafka Web changelog
 
+## 0.8.3 (Unreleased)
+- [Enhancement] Auto-qualify topics with DLQ/dlq/dead_letter_queue postfixes to DLQ view.
```

Comment:
I would change this to `Auto-qualify topics with dlq/dead_letter case insensitive postfixes to DLQ view`